### DISCUSSION TITLE
2.15.1

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#7706] [Announcements]
* Fix: Fix syntax when logging image rendition generation (Jake Howard)
 * Fix: Increase version range for django-filter dependency (Serafeim Papastefanos)
 * Fix: Prevent bulk action checkboxes from displaying on page reports and other non-explorer listings (Matt Westcott)
 * Fix: Fix errors on publishing pages via bulk actions (Matt Westcott)
 * Fix: Fix `csrf_token` issue when using the Approve or Unlock buttons on pages on the Wagtail admin home (Matt Westcott)

<hr /><em>This discussion was created from the release <a href='https://github.com/wagtail/wagtail/releases/tag/v2.15.1'>2.15.1</a>.</em>

**Comment by erwinfeser** [comment]
It seems to be that something in this release is not working in relation to image rendition generation 
<img width="849" alt="image" src="https://user-images.githubusercontent.com/567788/145235050-66d57c5b-9f23-4ef4-87d8-fb59e8d8e3da.png">

**Reply by gasman** [reply]
Are you sure you're running 2.15.1 rather than 2.15? This looks like an issue that was introduced in 2.15, and fixed in 2.15.1 - https://github.com/wagtail/wagtail/pull/7391#issuecomment-963677099, https://github.com/wagtail/wagtail/pull/7694

If you are running 2.15.1, please can you share the raw stack trace ('raw' from the menu in the top right) so we can investigate this further?
